src/main/java/frc/robot/commands/drivetrain/DriveStraight.java

```diff
@@ -3,36 +3,35 @@
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.wpilibj2.command.PIDCommand;
 import frc.robot.Robot;
-import frc.robot.subsystems.drivetrain;
+import frc.robot.subsystems.DriveTrain;
 
-public class driveStraight extends PIDCommand {
-    
-    private final drivetrain m_drive;
+public class DriveStraight extends PIDCommand {
+    private final DriveTrain drive;
```

Comment:
Good. As long as we use a consistent style. Some like naming member variables with a `m_` prefix, this way is good too. Often we just name the variable as the class name, with first character turned lowercase. Whichever way, consistency is key.